### DISCUSSION TITLE
Hi, it would be great if all depthmap addons supported Vector Displacement Maps

### DISCUSSION THREAD

**Discussion by ARandomUserFromGithub** [Extraltodeus/depthmap2mask#28] [Ideas]
Vector Displacement maps would allow to create creases behind perceived standing objects and characters, preventing a trail to draw behind them when visualised in Blender or a software capable of displaying parallaxes.

**Comment by Extraltodeus** [comment]
Would you have more informations on that subject?

**Comment by ARandomUserFromGithub** [comment]
Here is an example for Unreal Engine. From a simple set of textures, it could create the geometry of an ear onto a cube. https://forums.unrealengine.com/t/vector-displacement-map/4328
This page also speaks about vector displacement mapping https://forum.unity.com/threads/best-tool-asset-store-award-amplify-shader-editor-node-based-shader-creation-tool.430959/page-115
ZBrush have that capability too: http://docs.pixologic.com/user-guide/3d-modeling/exporting-your-model/vector-displacement-maps/
Vector displacement for Blender https://docs.blender.org/manual/en/latest/render/shader_nodes/vector/vector_displacement.html

The idea would be to generate more accurate parallax images, preventing a trail of pixels to draw behind standing subjects and objects.

Facebook's parallax api solves the problem with subjects by blurring behind.

I haven't found a nice way to preview parallax images on PC, yet. Looking Glass have a virtual looking glass, there's also an easy way to preview parallax images in Blender, but that doesn't remove the trail vortex behind standing models.

**Comment by Extraltodeus** [comment]
Would that extension do that?

https://github.com/thygate/stable-diffusion-webui-depthmap-script

**Reply by ARandomUserFromGithub** [reply]
I doubt so, as it generates only a height map, but not a crease map.
Height maps causes parallax pictures to draw standing subjects and objects draw a trace back to the back of the render. Example, a person would become a thick wall.
A crease map prevents this from happening, tho, I'm not too sure if there exist any program that could be used out the box right now for vector displacement. I'll ask on the project looking glass discord, they're experts in that stuff.
Blender, Unreal, Unity, and similar software are capable of handling those files nevertheless.

**Comment by ARandomUserFromGithub** [comment]
Hi @Extraltodeus 
I found this page for great raymarching examples 
https://iquilezles.org/demoscene/
